src/app/featureFlags.ts

```diff
@@ -4,6 +4,9 @@ import {EnvironmentID} from "../environments/environment.interface";
 const flagsForEnvironment = (environmentId: EnvironmentID) => {
   return {
     myDonationsEnabled: environmentId !== 'production',
+
+    // Friendly Captcha only in dev as we have to add support to identity service before enabling in staging.
+    friendlyCaptchaEnabled: environmentId === 'development',
```

Comment:
(in dev the captcha result appears not to be verified currently so sending the response from FriendlyCaptcha instead of recapcha doesn't block registration)

Reply:
I think that's only the case if you changed your local env to have `RECAPTCHA_BYPASS=1` @bdsl. I don't think we can assume it's true for everyone with that env.

Reply:
I don't have `RECAPTCHA_BYPASS=1` but I do have the key from here https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do set that allows all captchas to pass, same as we have in .env.example:

https://github.com/thebiggive/identity/blob/fc966a2fe4fe91351a7e6a708b11104195688ba7/.env.example#L11